data/language/dutch.txt

```diff
@@ -1709,7 +1709,7 @@ STR_1707    :Teveel werknemers in dit park
 STR_1708    :{SMALLFONT}{BLACK}Werkgebied van deze werknemer instellen
 STR_1709    :Werknemer ontslaan
 STR_1710    :Ja
-STR_1711    :{WINDOW_COLOUR_1}Weet je zeker dat je {STRINGID} wilt ontslaan?
+STR_1711    :{WINDOW_COLOUR_1}Weet je zeker dat je {STRINGID} wil ontslaan?
```

Comment:
Het kan inderdaad beide. Maar bij nader inzien wordt het zo wel erg informeel inderdaad. Ik stel voor dat we alles net wat formeler houden en voor wilt/doelstelling gaan.